app/components/list-item/index.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react'
+import ListItem from '.'
+import { ListItemType } from '@/app/app-types'
+
+describe('List-item component: test cases', () => {
+  const items: ListItemType[] = [
```

Comment:
Try to get the test data from faker

Reply:
I have used 'faker' for test data in the .test file. I'll update this component items data in my further implementation.